clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/services/RegionScopedProviderFactory.groovy

```diff
@@ -114,6 +114,14 @@ class RegionScopedProviderFactory {
       new DefaultLaunchConfigurationBuilder(getAutoScaling(), getAsgService(), getSecurityGroupService(), userDataProviders, localFileUserDataProperties, deployDefaults)
     }
 
+    LaunchTemplateService getLaunchTemplateService() {
+      return new LaunchTemplateService(amazonEC2)
+    }
+
+    AwsConfiguration.DeployDefaults getDeploymentDefaults() {
```

Comment:
renamed because groovy was compiling the method to call itself instead of returning the property. Thanks Groovy

Reply:
And was bombing silently